flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java

```diff
@@ -227,11 +241,77 @@ public class RocksDBNativeMetricOptions implements Serializable {
             ConfigOptions.key(METRICS_COLUMN_FAMILY_AS_VARIABLE_KEY)
                     .booleanType()
                     .defaultValue(false)
-                    .withDescription("Whether to expose the column family as a variable.");
+                    .withDescription(
+                            "Whether to expose the column family as a variable for RocksDB property based metrics.");
+
+    // --------------------------------------------------------------------------------------------
+    //  RocksDB statistics based metrics, report at database level
+    // --------------------------------------------------------------------------------------------
+
+    public static final ConfigOption<Boolean> MONITOR_BLOCK_CACHE_HIT =
```

Comment:
Could we also create a new class called "RocksDBStatistics" and use it just like "RocksDBProperty" ?
Maybe it could be clearer to me. WDYT?

Reply:
To be honest, I think previous `RocksDBProperty` is a bit complex, and I prefer to use the full metrics name in the source code as it could be easily searched.